### DISCUSSION TITLE
Is there a way to change validationSchema dinamically?

### DISCUSSION THREAD

**Discussion by tjdam** [react-hook-form/react-hook-form#2206] [Q&A] [answered]
I was wondering if it's possible to swap different validationSchema after useForm has already been called?
Thanks :)

**Comment by bluebill1049** [comment, marked as answer]
you can use resolver and hoist different schema inside.

**Comment by tjdam** [comment]
I'm probably being super noob here, but you mean resolver as in the replacement for validationSchema in RHF 6?

**Reply by bluebill1049** [reply]
you are not, yes in v6, and you can do the same in v5 in valdiationResolver
